### DISCUSSION TITLE
Removing the last subtitle line with CLI /convert ?

### DISCUSSION THREAD

**Discussion by rRobis** [SubtitleEdit/subtitleedit#8391] [Q&A] [answered]
I was trying to find, if there is a way to delete last subtitle line when doing convert?

I would find this useful, when I know, that the last line contains text that is used as a visual title for a video (e.g. for shorts).

The titles are always different, and they don't contain something specific, so that I could use /replacecontains

Otherwise, when I'm uploading the subtitles somewhere, I have to manually delete the title from the subtitles.

I probably could write some script to find the last gap and remove until end of file, but it would add unnecessary complexity to the process.

**Comment by niksedk** [comment]
It's in the batch convert atm:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/def9eb7a-230f-4255-a16a-b89774a45b2e)

**Reply by rRobis** [reply]
Is it possible to do this from CLI, or at this point it's not possible?

**Comment by niksedk** [comment]
Available in CLI in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.5/SubtitleEditBeta.zip
Will now run in order, and arguments can be added multiple times.

**Reply by rRobis** [reply]
I posted another test comment on the commit

**Reply by niksedk** [reply, marked as answer]
Thx for testing - better now?
https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.5/SubtitleEditBeta.zip

**Reply by rRobis** [reply]
Yes, it's working now correctly. Thank you very much 😄
